aviatrix/resource_aviatrix_aws_tgw.go

```diff
@@ -141,6 +141,13 @@ func resourceAviatrixAWSTgw() *schema.Resource {
 					},
 				},
 			},
+			"cloud_type": {
```

Comment:
Should this be `ForceNew`? If not, do we need to modify the Update function as well?

Reply:
If this was ForceNew, existing TGWs would be deleted; need to be mindful of backward-compatibility, though I totally understand why ForceNew would be useful here.

Reply:
Yes, I added a commit to block updating cloud_type.